### PR TITLE
added subcategory

### PR DESCRIPTION
**Pull request by tejashreee03**
closes issue #2158 

please merge this pull request @CBID2

### CODE REVIEW

File: database/resources/documents.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+      "name": "Official Doc of TypeScript",
+      "description": "The comprehensive resource for TypeScript, offering in-depth documentation, tutorials, and reference materials to help developers in their JavaScript projects.",
```
Comment (CBID2) [inline database/resources/documents.json:4]:
```suggestion
      "description": "The comprehensive resource for TypeScript, offering in-depth documentation, tutorials, and reference materials to help developers with their JavaScript projects.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Your PR is off to a great start @tejashreee03. It just needs something revision. Also, when filling out your pull request's form, please follow the template's instructions.

**Comment by tejashreee03** [comment]
created new pr

**Comment by Anmol-Baranwal** [comment]
@CBID2 
He created two PRs, we can go ahead with that. I'm closing this.

**Pull request closed without merging**